### PR TITLE
Login/Create account with GraphQL

### PR DESCRIPTION
**Pull request by StefanPahlplatz**

### CODE REVIEW

File: frontend/src/modules/feed/components/UserDropdown/UserDropdown.js [hunk 1]
```diff
@@ -39,18 +36,18 @@ class DropdownRemote extends Component {
...
-    axios.get(settings.API_USER_SEARCH).then(resp => {
-      const options = [];
-      resp.data.forEach(item => {
-        options.push({
-          text: `${item.first_name} ${item.last_name}`,
-          value: item.id,
-          image: { avatar: true, src: item.avatar_url },
-        });
-      });
+    // axios.get(settings.API_USER_SEARCH).then(resp => {
```
Comment (MvanStraten) [inline frontend/src/modules/feed/components/UserDropdown/UserDropdown.js:39]: More comments

File: frontend/src/App.js [hunk 2]
```diff
@@ -28,9 +29,9 @@ const Routes = [
   <StatisticsPage path={routes.PATH_STATISTICS} />,
   <SettingsPage path={routes.PATH_SETTINGS} />,
   <AddTransactionPage path={routes.PATH_ADD_TRANSACTION} />,
-  <ResetPasswordPage path={routes.PATH_RESET_PASSWORD} />,
-  <ChooseTeamPage path={routes.PATH_CHOOSE_TEAM} />,
-  <CommentPage path="/comments/:comment" />,
+  // <ResetPasswordPage path={routes.PATH_RESET_PASSWORD} />,
+  // <ChooseTeamPage path={routes.PATH_CHOOSE_TEAM} />,
+  // <CommentPage path="/comments/:comment" />,
```
Comment (MvanStraten) [inline frontend/src/App.js:34]: Comments

**Review by MvanStraten — COMMENTED** [review]
Just some comments, looks good!

File: frontend/src/modules/feed/components/UserDropdown/UserDropdown.js (hunk 1 above)
Comment (StefanPahlplatz) [reply frontend/src/modules/feed/components/UserDropdown/UserDropdown.js:39]: Temporary non functional code until the features are implemented in GraphQL.

File: frontend/src/App.js (hunk 2 above)
Comment (StefanPahlplatz) [reply frontend/src/App.js:34]: Temporary non functional code until the features are implemented in GraphQL.

File: frontend/src/modules/feed/components/UserDropdown/UserDropdown.js [hunk 3]
```diff
@@ -39,18 +36,18 @@ class DropdownRemote extends Component {
     this.setState({ isFetching: true });
     console.log(searchQuery);
```
Comment (thnukid) [inline frontend/src/modules/feed/components/UserDropdown/UserDropdown.js:37]: Debug command?

**Pull request merged**